hack/kops-patch.yaml

```diff
@@ -0,0 +1,145 @@
+spec:
```

Comment:
this can be left as empty file, i assume it was copy/pasted from ebs so it's not needed by us

Reply:
Looks like I can use it to patch the cluster file with the authenticator setup.